script/options.js

```diff
@@ -46,23 +44,22 @@ function applyDefaultOptions(options) {
     redirects: [],
   });
 
-  if (options.buildtype === undefined) {
+  if (!options.buildtype) {
     options.buildtype = environments.DEVELOPMENT;
+    options.protocol = 'http';
+  } else {
+    options.host = hostnames[options.buildtype];
+    options.protocol = 'https';
   }
+
+  options.hostUrl = `${options.protocol}://${options.host}${
+    options.port ? `:${options.port}` : ''
+  }`;
 }
```

Comment:
In working through where to derive the `hostUrl` property and deciding what to name it, I realized that there is a discrepancy between what host/hostname means in the browser versus in Node, so the logic touched more than I wanted, but I think it makes sense. 

- No default value for `buildtype` from the command line. This is so we can detect a dev environment (dev.va.gov) versus localhost.
    - If no buildtype, we default to http, development, and use the host as passed from the command line.
    - If there is a buildtype, we use https and override the host to derive it from the buildtype.

Reply:
Update - add an override on the port as well